eduNEXT/tvm
# Tests/tvm commands

Description
========
This PR adds tests to migrated commands

Migrated commands:
- list_versions
- install
- uninstall
- use
- install_plugin
- uninstall_plugin

How to test
========
1. Use this branch (tests/tvm_commands)
2. Create and run a virtual environment
3. Run ``make test``
4. You should see a summary with all tests passed.
